### DISCUSSION TITLE
Remove the minimum window size limit or make it configurable by the hyprland config

### DISCUSSION THREAD

**Discussion by ingenarel** [hyprwm/Hyprland#6920] [General]
i tried asking about this in the hyprland discord and a user said that this feature doesn't exist and to make a feature request for it. i also couldn't find something like this in the internet. but i did find this open issue tho https://github.com/hyprwm/Hyprland/issues/4414 

![image](https://github.com/user-attachments/assets/a880db03-bf8a-4b42-b59d-9ee8d32ae618)

i have a small monitor and for most of the times, i like to keep a process on the terminal like this by putting the terminal on the top.  this connects to a render farm's server. 
however, because i have a small monitor i want to keep it as much small as i can because then i can work freely while also keeping an eye on what's happening.

yes ik that windows like firefox gets compressed. but this happens even when the size limit is on...
![image](https://github.com/user-attachments/assets/851f3010-f135-4a7c-83bc-266f03d623a0)

that's why i'm asking to either remove this completely, or make it so it's configurable by the hyprland config

**Comment by ghost** [comment]
Have you considered making multiple special workspaces for each of your "minimized" windows instead of stacking them up in your work area? I also have a tiny laptop, and special workspaces have turned out to be a lifesaver. I have one for each window I need frequent access to and just pop them in/out with a keybind. This has been a lot more efficient for me than having to resize windows/mess up my layout, and it freed up crucial screen space. 

I have mine configured to pop up over my active workspace at 70% of the screen size with no blur/dim so I can still see what's going on in each space. And it only takes a quick keybind to move windows to/from if needed. Maybe a similar workflow would work for you too.

**Reply by ghost** [reply]
Another option that might be even better for what you're specifically trying to do is using a kitten panel to display a single line of the output you're watching at the top of your screen. It's thinner than any window would be and you can make it display pretty much anything you want.

**Comment by azizsayfiddin** [comment]
Same.
Telegram has a minimum window size, and I want to be able to change its window size without minimum size limitations.
